### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by mbonamensa** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#34] [Outreachy 2022] [answered]
Hi,
My name is Maame ***(maa-meh)***. I am an Outreachy Applicant and I am happy to learn and improve my skills while adding value and contributing to this Open Science project.

**Comment by jesutobi** [comment, marked as answer]
> Hi, My name is Maame _**(maa-meh)**_. I am an Outreachy Applicant and I am happy to learn and improve my skills while adding value and contributing to this Open Science project.

welcome @mbonamensa , you can tag @BatoolMM or @alswajiab  when you try to state the #issue you want to solve, they are our mentors ,.after that you can clone the repo and start working...if you have issues with creating a pull request or cloning the repo just tag me, i am willing to help

**Comment by BatoolMM** [comment]
Thank you @jesutobi for helping out new interns!

**Reply by jesutobi** [reply]
you are welcome
